### ISSUE TITLE
Batchsize decreasing with each new GPU

### ISSUE THREAD

**Issue by DiegoJota** [NVIDIA/NeMo#2988]
Hi,

With every new GPU I add I find I'm forced to lower batchsize because of memory overflows. This is normal? 
For example with 1 GPU I can use 68 BatchSize, with 2 GPUs I can use 67 and with 5 I can use 62.

OS:
Ubuntu 18.04
NeMo 1.3.0
Python 3.8

**Comment by titu1994** [comment]
No, we use same batch size across all GPUs, and keep consistent 32 batch size across all. DDP should not normally need to reduce batch size with more gous, as long as all GPUs are of the same type and have same memory
